lib/Raven/Serializer.php

```diff
@@ -109,6 +109,12 @@ protected function serializeValue($value)
         if (is_null($value) || is_bool($value) || is_float($value) || is_integer($value)) {
             return $value;
         } elseif (is_object($value) || gettype($value) == 'object') {
+            // If the value is a callable and expects no parameters, we evaluate the
+            // callable value by calling it.
+            if (is_callable($value) && Raven_Util::getCallableParamNum($value) === 0) {
+                return $value();
```

Comment:
Although the concept of lazy-loading is indeed nice, you can't base your assumptions on the fact that any callable/method should be called. It's not something related to security issues, it's just common sense. What if the method calls the database to do some heavy query to get results? Also (I'm not really sure what I'm going to say makes sense), usually Sentry is called when an error happens, and this means that there could be cases where the application is broken and if the method to call relies on something that crashes then bad things can happen